src/python/WMCore/WebTools/DBSRESTFormatter.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python
+"""
+_DBSRESTFormatter_
+
+A basic REST formatter. The formatter takes the data from the API call, turns it into the
+appropriate format and sets the CherryPy header appropriately.
+
+Could add YAML via http://pyyaml.org/
+"""
+from __future__ import (division, print_function)
+from WMCore.WebTools.RESTFormatter import RESTFormatter
+
+class DBSRESTFormatter(RESTFormatter):
+    def __init__(self, config):
+	RESTFormatter.__init__(self, config)
```

Comment:
a) indentation is wrong
b) self is not needed. You will get an error the first time you try to use this.